spec/queries/action_page_filters_spec.rb

```diff
@@ -0,0 +1,48 @@
+require "rails_helper"
+
+describe ActionPageFilters do
```

Comment:
these tests aren't pretty, but testing this new object requires detailed data. 

I'm not sure if there are any good ways to clean them up, but it would be nice to do that.